<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-8LQ1C4T7" failures="0" tests="4" name="com.bookswagon.qa.testcases.LoginPageTest" time="9.500" errors="1" timestamp="14 Mar 2023 07:20:25 GMT" skipped="3">
  <testcase classname="com.bookswagon.qa.testcases.LoginPageTest" name="setup" time="9.500">
    <error message="Cannot invoke &quot;org.openqa.selenium.WebDriver.manage()&quot; because &quot;com.bookswagon.qa.base.TestBase.driver&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.manage()" because "com.bookswagon.qa.base.TestBase.driver" is null
at com.bookswagon.qa.base.TestBase.initialization(TestBase.java:47)
at com.bookswagon.qa.testcases.LoginPageTest.setup(LoginPageTest.java:24)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setup -->
  <testcase classname="com.bookswagon.qa.testcases.LoginPageTest" name="loginPageTitleTest" time="0.000">
    <skipped/>
  </testcase> <!-- loginPageTitleTest -->
  <testcase classname="com.bookswagon.qa.testcases.LoginPageTest" name="bookswagonLogoTest" time="0.000">
    <skipped/>
  </testcase> <!-- bookswagonLogoTest -->
  <testcase classname="com.bookswagon.qa.testcases.LoginPageTest" name="loginTest" time="0.000">
    <skipped/>
  </testcase> <!-- loginTest -->
</testsuite> <!-- com.bookswagon.qa.testcases.LoginPageTest -->
